boards/xtensa/nxp_adsp_imx8x/nxp_adsp_imx8x-pinctrl.dtsi

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/imx8qxp-pinctrl.h>
+
+&iomuxc {
+	/omit-if-no-ref/ iomuxc_uart2_rx_uart2_rx: IOMUXC_UART2_RX_UART2_RX {
+		pinmux = <SC_P_UART2_RX IMX8QXP_DMA_LPUART2_RX_UART2_RX
+			SC_PAD_CONFIG_NORMAL SC_PAD_ISO_OFF>;
+	};
+
+	/omit-if-no-ref/ iomuxc_uart2_tx_uart2_tx: IOMUXC_UART2_TX_UART2_TX {
```

Comment:
omit-if-no-ref is meant for autogenerated files, why is this defined here?

Reply:
Not sure why I decided to keep the `/omit-if-no-ref/` statements as they're not necessary at this point. Either way, I've removed them.